client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -378,21 +382,21 @@ public void testCreateIndex() throws IOException {
             request = new CreateIndexRequest("twitter6");
             // tag::create-index-whole-source
             request.source("{\n" +
-                    "    \"settings\" : {\n" +
```

Comment:
Can you revert the formatting change? I think the old way is better. If we want to do it we should do it in a separate PR anyway.